src/utils/call-responses.ts

```diff
@@ -0,0 +1,58 @@
+import { Request, Response } from 'express';
+
+import { AppCallResponse, AppForm } from '../types';
+import { AppCallResponseTypes } from '../constant';
+
+export type FieldValidationErrors = {[name: string]: string};
+
+export type CallResponseHandler = (
+    req: Request,
+    res: Response & {
+        json: (
+            callResponse: AppCallResponse
+        ) => void,
+    }
+) => Promise<void>;
```

Comment:
Optional

In reference to me previous comment, you also need to do this to tell typescript what you will receive and what you should respond with:

```suggestion
export type CallResponseHandler = (
    req: Request<unknown, AppCallResponse, AppCallRequest>,
    res: Response<AppCallResponse>
) => Promise<void>;
```

I also had to add this to make eslint approve of this type. This was required with the current version of this type as well.

```json
        "@typescript-eslint/no-unused-vars": ["warn"],
```